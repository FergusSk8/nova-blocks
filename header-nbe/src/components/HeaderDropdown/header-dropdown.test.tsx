import React from "react";
import { shallow } from "enzyme";

import HeaderDropdown from "./index";

describe("Header Dropdown", () => {
  let wrapper = shallow(<HeaderDropdown />);

  beforeEach(() => {
    wrapper = shallow(<HeaderDropdown />);
  });

  test("should show correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
