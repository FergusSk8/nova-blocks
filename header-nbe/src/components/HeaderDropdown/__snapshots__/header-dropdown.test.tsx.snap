// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Dropdown should show correctly 1`] = `
<Dropdown
  mouseEnterDelay={0.15}
  mouseLeaveDelay={0.1}
  overlayClassName=""
/>
`;
